src/Redux/Reducers/noteReducer.js

```diff
@@ -15,6 +15,21 @@ export default function (state = initialState, action) {
     case actions.SORT_NOTE:
       const sortedVal = state.sorted;
       return { ...state, sorted: !sortedVal };
+    case actions.UPDATE_NOTE:
+      const newNote = action.newNote;
+      return {
+        ...state,
+        value: state.value.map((note) =>
+          note.id === newNote.id
+            ? {
+                ...note,
+                name: newNote.name,
+                date: newNote.date,
+                note: newNote.note,
```

Comment:
Is this a new note? or are you updating an existing note?

consider changing the name of the variable to something more appropriate to the action of this case. Could be `updatedNoteValues` as an example.